tests/trustless_gateway_car_test.go

```diff
@@ -374,6 +384,9 @@ func TestTrustlessCarDagScopeAll(t *testing.T) {
 }
 
 func TestTrustlessCarEntityBytes(t *testing.T) {
+	tooling.LogTestGroup(t, GroupTrustlessGateway)
+	tooling.LogIPIP(t, IPIP402, "4.2.2")
```

Comment:
We can associate an ipip + it's section to a group of tests.

Reply:
IPIPs aren't, in general, that long. I'm not sure if including their section is useful, as they all have more or less the same structure (summary, motivation, design, rationale, etc).